enterprise/server/test/integration/remote_execution/rbetest/rbetest.go

```diff
@@ -699,6 +682,18 @@ func (r *Env) AddBuildBuddyServerWithOptions(opts *BuildBuddyServerOptions) *Bui
 	server := newBuildBuddyServer(r.t, env, opts)
 	r.buildBuddyServers[server] = struct{}{}
 	r.updateAppProxy()
+
+	grpcAddress := fmt.Sprintf("grpc://localhost:%d", server.GRPCPort())
+	u, err := url.Parse(grpcAddress)
+	require.NoError(t, err)
+
+	// Note: flags.Set registers a t.Cleanup function to clean up flags. Make sure
+	// that happens after the servers shut down and are no longer using the flags
+	flags.Set(t, "app.events_api_url", *u)
+	flags.Set(t, "app.cache_api_url", *u)
```

Comment:
instead of pointing these at an individual app, you can point them at the app load balancer address (`Env.AppProxy.GRPCTarget()`). I think we could do that in `NewRBETestEnv()` right after we create the load balancer, and then hopefully that would let us revert the other changes to how we're doing cleanup?

Reply:
Nice - that simplifies things a lot